src/ipa/ipu3/ipu3.cpp

```diff
@@ -600,7 +600,7 @@ void IPAIPU3::fillParams(unsigned int frame, ipu3_uapi_params *params)
  */
```

Comment:
"Explicit the usage" doesn't make sense.

I assume you mean..

"""
ipa: ipu3: Explicitly use the statistics parameter
"""

s/is really used/are really used/